student/admin.py

```diff
@@ -1,3 +1,10 @@
-# from django.contrib import admin
+from django.contrib import admin
+from django.apps import apps
 
-# Register your models here.
+# Register your models here. # Register your models here.
+models = apps.get_models()
```

Comment:
it takes 2 lines to register the models you have and 6 to write this loop which I'm not sure you understand. I know Sim showed it, but I think having simpler code here is better.

Reply:
@ifireball Can you check it now?